File: benchmark/results/java-multilang/run-003/b-baseline/output/com/metaengine/demo/inventory/services/ReservationService.java
package com.metaengine.demo.inventory.services;

import com.metaengine.demo.inventory.aggregates.StockItem;
import java.util.List;

/** ReservationService domain service for the inventory domain. */
public class ReservationService {
    /** Create a new StockItem from a partial input. */
    public StockItem create(StockItem input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a StockItem by its identifier. May return null. */
    public StockItem findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List StockItems up to the given limit. */
    public List<StockItem> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a StockItem by its identifier. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-inline-sonnet/run-005/b-baseline/output/src/domain/inventory/services/ReservationService.ts
import { StockItem } from "../aggregates/StockItem";

/** Service providing stock reservation operations for inventory. */
export class ReservationService {
  create(input: Partial<StockItem>): StockItem {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): StockItem | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): StockItem[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
